staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -418,7 +418,8 @@ func (o *DebugOptions) visitPod(ctx context.Context, pod *corev1.Pod) (*corev1.P
 
 // debugByEphemeralContainer runs an EphemeralContainer in the target Pod for use as a debug container
 func (o *DebugOptions) debugByEphemeralContainer(ctx context.Context, pod *corev1.Pod) (*corev1.Pod, string, error) {
-	klog.V(2).Infof("existing ephemeral containers: %v", pod.Spec.EphemeralContainers)
+	logger := klog.FromContext(ctx)
+	logger.V(2).Info("existing ephemeral containers", pod.Spec.EphemeralContainers)
```

Comment:
The entire PR has one major problem: lack of keys in the key/value pairs. This is just one example.

@yanjing1104: have a look again at https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments.

You can run `hack/verify-golangci-lint.sh` on your machine and it should complain about the modified log calls.

Reply:
Fixed the complaints after running the script, thank you. :D